samples/instrumentation-quickstart/app.py

```diff
@@ -31,6 +35,27 @@
 from flask import Flask, url_for
 from opentelemetry.instrumentation.flask import FlaskInstrumentor
 
+LoggingInstrumentor().instrument()
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+logHandler = logging.StreamHandler()
+formatter = jsonlogger.JsonFormatter(
+    "%(asctime)s %(levelname)s %(message)s %(otelTraceID)s %(otelSpanID)s %(otelTraceSampled)s",
+    rename_fields={
+        "levelname": "severity",
+        "asctime": "timestamp",
+        "otelTraceID": "logging.googleapis.com/trace",
+        "otelSpanID": "logging.googleapis.com/spanId",
```

Comment:
Isn't there an OTel spec for trace and span ID that doesn't match this?

I understand what you have here works with most GCP agents, but it seems suboptimal to have an OTel sample use a legacy GCP-specific key when there's a generic alternative.

Reply:
Yes.  The spec is currently experimental: https://opentelemetry.io/docs/specs/otel/compatibility/logging_trace_context/.  We are separately working on getting those keys supported by agents.  Once that support exists, we will update the samples to use those keys.